### ISSUE TITLE
 key assignment

### ISSUE THREAD

**Issue by MuzanRgc** [huskcasaca/effortless-fabric#6]
Hi, I downloaded the mod and when I open minecraft and look for a way to assign a key to use the mod but I do not see any option to assign it and I get the default "Alt" key but that key is already in use and no matter how hard I look I can not find the option, how can I assign the key I want or is not yet available key assignment?

**Comment by DemonSlayer112** [comment]
Would also like to see this feature added.
Constantly accidently changing modes when I'm just alt tabbing.

**Comment by JJSax** [comment]
Agreed.  It conflicts with other mods where I need this button more, so I'd prefer moving this one to a harder to reach button.

**Comment by LucasAiden** [comment]
yes my friend wants to rebind to something else

**Comment by huskcasaca** [comment]
Added in 1.4.1. Try the latest build.
